chargeio/models/Credit.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using ChargeIo.Infrastructure;
+using ChargeIO;

```

Comment:
There's an inconsistency between the re-added IPaymentMethod in the (original) namespace ChargeIO, and the changed classes, which are all ChargeIo. I would recommend reverting the changed files (with the trailing lowercase 'o') to the original uppercase 'O', which also maintains compatibility for any consumers.